src/isar_turtlebot/turtlebot/taskhandlers/taskhandler.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC, abstractmethod
+
+from isar_turtlebot.models.turtlebot_status import Status
+
+
+class TaskHandler(ABC):
+
+    """Baseclass for task handlers"""
+
+    @abstractmethod
+    def start(self, task_input) -> None:
+        pass
+
+    @abstractmethod
+    def get_status(self) -> Status:
+        pass
```

Comment:
```suggestion

    @abstractmethod
    def get_status(self) -> Status:
        pass
        
    @staticmethod
    def _get_status_from_message(message: dict) -> Status:
        status_code: int = message["status_list"][0]["status"]
        if status_code == 1:
            return Status.Active
        elif status_code == 2 or status_code == 4:
            return Status.Failure
        elif status_code == 3:
            return Status.Succeeded
        else:
            return Status.Unexpected
```

Reply:
This is a suggestion for how to make this method common across the handlers, but there may be better ways. Another option would be to have it as a simple method in this file and just import it from the other task handlers.